compiler/packages/babel-plugin-react-compiler/src/HIR/Globals.ts

```diff
@@ -136,6 +136,17 @@ const TYPED_GLOBALS: Array<[string, BuiltInType]> = [
           returnValueKind: ValueKind.Primitive,
         }),
       ],
+      [
+        'min',
```

Comment:
whole bunch more `Math.*` stuff missing.

Reply:
Makes sense, added min as `Math.min` and `Math.max` differences surprised me a few times. Will go ahead and add a few others